domino_cost/cost_dashboard.py

```diff
@@ -76,7 +87,17 @@
                     ),
                     width=2,
                 ),
-                dbc.Col(width=9),
+                dbc.Col(
+                    dcc.DatePickerRange(
+                        id="date-picker-range",
+                        min_date_allowed=date(2023, 1, 1),
+                        max_date_allowed=today,
+                        initial_visible_month=last_30,
```

Comment:
Nit: I would select the month related to today (for example if I click now, it's showing me August instead of September, and it would keep showing me August up until the 31st of September I guess, it's weird)

Reply:
I set it for last 30 days to align with the default 30 days that we have set, but that makes sense.